core/src/main/java/org/bitcoinj/script/ScriptPattern.java

```diff
@@ -198,7 +198,7 @@ public static boolean isP2WSH(Script script) {
 
     /**
      * Returns whether this script matches the format used for multisig outputs:
```

Comment:
Perhaps we should use the tern "M-of-N multisig" here? This would define what m and n actually means.

And the Javadoc change could go on its own commit, as it's not related to the exception change as far as I understand.

Reply:
https://github.com/bitcoinj/bitcoinj/pull/1867